stable/redis-ha/values.yaml

```diff
@@ -60,19 +54,13 @@ persistentVolume:
   ## Default: volume.alpha.kubernetes.io/storage-class: default
   ##
   storageClass: ""
-
   ## Subdirectory of redis data Persistent Volume to mount
   ## Useful if the volume's root directory is not empty
   ##
   subPath: ""
-
 ## Redis image version
-redis_image: gcr.io/google_containers/redis:v1
-
-
-
+redis_image: quay.io/smile/redis:4.0.2
```

Comment:
Can you add the source of this image in the sources of the Chart.yaml?

Reply:
Done!